lib/mixlib/install/backend/base.rb

```diff
@@ -195,6 +221,26 @@ def normalize_architecture(architecture)
             architecture
           end
         end
+
+        private
+
+        #
+        # Custom map Chef's supported windows desktop versions to the server versions we currently build
+        # See https://docs.chef.io/platforms.html
+        #
+        def map_custom_windows_desktop_versions(desktop_version)
+          unless SUPPORTED_WINDOWS_DESKTOP_VERSIONS.include?(desktop_version)
+            raise UnsupportedVersion, "Unsupported Windows desktop version `#{desktop_version}`. Supported versions: #{SUPPORTED_WINDOWS_DESKTOP_VERSIONS.join(", ")}."
+          end
+
+          server_version = Util.map_windows_desktop_version(desktop_version)
+
+          # Windows desktop 10 officially maps to server 2016.
+          # However, we don't build server 2016 at this time, so we default to 2012r2
```

Comment:
I also updated this PRs description to refer to _testers_ as opposed to _builders_.